spec/lita/adapters/slack/api_spec.rb

```diff
@@ -9,6 +9,7 @@
 
   before do
     config.token = token
+    config.postMessage = {}
```

Comment:
I had to do this because configuration is sticky for some reason, once my new spec runs, other specs fail with `link_names: 1` in them.